### PR TITLE
New Lawyer Suits 

### PR DESCRIPTION
**Pull request by Doritodude77**
About the PR
Redesigns and expands on the existing selection of Lawyer Suits,and updates the LawDrobe to dispense them.
Changes include: 
-Added pinstriped versions of the Blue and Black Lawyer Suits
-Added a Brown Suit and Green Suit
-Replaced Purple Suit with a design more consistent with the other preexisting jumpsuits
-Added a Lawyer Badged variant of the preexisting Journalist jumpsuit
-Redid Lawyer Jumpskirt skirts so that they no longer go over the suit's jacket in an unrealistic way
-Added Office/Serious skirt variations of Jumpskirt sets

Why / Balance
I wanted some more stylistic variation.

Media
Preview1Resize

(As Visual Studio was being annoying, I am currently unable to provide non-sprite preview images. The following picture was taken by PursuitInAshes before the latest fix.) 

UristPreview

Changelog
:cl:

Realizing there was a bit more room for variation that what was already there, one of the more design-oriented NT lawyers negotiated an affiliate deal with a fashion designer to have the LawDrobes stocked with some new outfits for everyone. 
Changes include:
add: Added pinstriped versions of the Blue and Black Lawyer Suits
add: Added a Brown Suit and Green Suit
tweak: Replaced Purple Suit with a design more consistent with the other preexisting jumpsuits
add: Added a Lawyer Badged variant of the preexisting Journalist jumpsuit
fix: Lawyer Jumpskirt skirts no longer go over the suit's jacket in an unrealistic way
add: Added Office/Serious skirt variations of Jumpskirt sets

Special thanks to the Cosmatic Drift discord, for design feedback and teaching assistance.

### CODE REVIEW

**Comment by UbaserB** [comment]
can i get some showcases

**Comment by potato1234x** [comment]
the old purple design looked better imo

**Comment by beck-thompson** [comment]
They look good to me! I actually like the purple one a lot. I would love a video or something so I can get a better look at them in game.

**Comment by Yousifb26** [comment]
I also prefer the old purple suit

**Comment by Doritodude77** [comment]
It's ready to go now!

**Comment by Golinth** [comment]
I think these look great, amazing job

**Comment by Doritodude77** [comment]
> This pull request has conflicts, please resolve those before we can evaluate the pull request.

Updated the conflicting lawdrobe content file to include the changes made in the other conflicting pull. No longer conflicts

**Comment by lzk228** [comment]
bloat

**Comment by chromiumboy** [comment]
If you have a concern @Izk228, please state it in full

**Comment by lzk228** [comment]
> If you have a concern @Izk228, please state it in full

most of maintainers are against bloating the game with a lot of same sprites but just only with different color

**Comment by Doritodude77** [comment]
I have gone over jumpskirts.yml and fixed the merge conflict- But to be honest about it, all I did was remove the conflict markers as no entries had been removed.

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml [hunk 1]
```diff
@@ -876,3 +876,14 @@
       sprite: Clothing/Uniforms/Jumpskirt/lawyergreenaltskirt.rsi
     - type: Clothing
       sprite: Clothing/Uniforms/Jumpskirt/lawyergreenaltskirt.rsi
+
+  parent: ClothingUniformSkirtBase
+  id: ClothingUniformJumpskirtOldDress
+  name: old dress
+  description: A worn-looking dress from a very long time ago.
+  components:
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:884]: you forgot `-type: entity`

**Comment by Flareguy** [comment]
Thank you for your contribution, but I'll be closing this PR for a few reasons:
1. We really need guidelines on clothing. Normally this isn't a huge problem but, with something that adds *this* much, I can't just merge this in good faith.
2. All of these are basic recolors and should probably just be a part of some new set of color suits / these need to be at least greyscaled.
3. The review I gave this PR earlier in discord probably still mostly applies. These are building on sprites that need to be replaced.

**Comment by Doritodude77** [comment]
> Thank you for your contribution, but I'll be closing this PR for a few reasons:
> 
> 1. We really need guidelines on clothing. Normally this isn't a huge problem but, with something that adds _this_ much, I can't just merge this in good faith.
> 2. All of these are basic recolors and should probably just be a part of some new set of color suits / these need to be at least greyscaled.
> 3. The review I gave this PR earlier in discord probably still mostly applies. These are building on sprites that need to be replaced.

Your 2nd contention has several problems;
A grayscaling of a single suit template not only nullifies all possible built-in tie color combinations (likely why the originals were not grayscaled), but also eliminates the possibility of patterns, as exemplified by the 2 pinstriped designs, 
And to blanket state that "all of these are basic recolors" not only ignores the above facts, but also forgets that the new suitskirt designs both repair a glaring design problem previously missed by the original, 
and introduce an additional, new skirt design not previously implemented (which was chosen to be the best out of 6 alternatives by the people surveyed). The reason provided completely ignores this.

And as also conveyed in discord, most current end users were fine and used to the sprites to begin with, making them adequate to be built upon (as evidenced by their not having been replaced by a theoretically perfect version earlier). With all due respect, the problems carried in your opinion are dwarfed by much worse offenders- It's not like they're completely unshaded and outlined in black!

**Comment by mirrorcult** [comment]
you can just have a template with multiple layers to color if you want differently colored ties

**Comment by Doritodude77** [comment]
At a _minimum_, it's a fix and some new variations, grayscaled or not

**Pull request closed without merging**